enum.rb

```diff
@@ -121,6 +127,7 @@ def my_map
   end
 
   def my_inject(proc = nil, ope = nil)
+    return inject(proc) if ope == nil?
     return inject(proc, ope) unless ope.nil?
```

Comment:
As other previous reviews have pointed out, we should handle cases where `my_inject` does not accept `block`. See the illustration below as the current method is raising a `LocalJumpError` error.

```ruby
[1,2,3,4,5].inject(:+)
#=> 15
irb(main):004:0> [1,2,3,4,5].my_inject(:+)
LocalJumpError (no block given (yield))
```